app/electron/main.ts

```diff
@@ -295,13 +296,17 @@ function startElecron() {
     mainWindow.webContents.on('dom-ready', () => {
       const octokit = new Octokit();
 
-      async function fetchLatestRelease() {
-        const response = await octokit.request('GET /repos/{owner}/{repo}/releases/latest', {
+      async function fetchRelease() {
+        const githubReleaseURL = `GET /repos/{owner}/{repo}/releases/latest`;
+        const response = await octokit.request(githubReleaseURL, {
           owner: 'kinvolk',
           repo: 'headlamp',
         });
-
-        if (response.data.name !== appVersion) {
+        if (
+          semver.gt(response.data.name, appVersion) &&
+          !response.data.name.startsWith('headlamp-helm') &&
+          !process.env.FLATPAK_ID
```

Comment:
I am fine having this here but you have a commit where it mentions not showing it for Flatpak.
So maybe remove the flatpak dedicated commit and add a code comment here to make it more clear what this does.

Reply:
done